docs/pagination.rst

```diff
@@ -58,10 +58,10 @@ The following Jinja macro renders a simple pagination widget.
 .. code-block:: jinja
 
     {% macro render_pagination(pagination, endpoint) %}
-      <div class=page-items>
+      <div class="page-items">
         {{ pagination.first }} - {{ pagination.last }} of {{ pagination.total }}
       </div>
-      <div class=pagination>
+      <div class="pagination">
```

Comment:
This change is not needed, undo this.

Reply:
Why it's not needed? It should be the class name.

Reply:
quotes are optional for attributes without spaces  in the value

Reply:
Didn't konw that, but it's best practice to not do that, right?

Reply:
Please change this.

Reply:
updated.